.github/workflows/ci.yml

```
@@ -31,7 +36,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x, 20.x, 21.x]
+        node-version: [21.x]
```

would recommend running these against node 20 as it's an LTS release and less prone to issues hitting their release channel

```suggestion
        node-version: [20.x]
```